src/EDF.jl

```diff
@@ -96,13 +96,16 @@ end
 """
     EDF.Header
 
-Type representing the header record for an EDF file.
+Type representing the header record for an EDF file. `EDF.File`s parsed from
+files which are not EDF+ compliant do not specify if data records are contiguous;
+data records in these files are considered contiguous by default.
 
 ## Fields
 
 * `version` (`String`): Version of the data format
 * `patient` (`String` or `EDF.PatientID`): Local patient identification
 * `recording` (`String` or `EDF.RecordingID`): Local recording identification
+* `continuous` (`Bool`): If true, data records are contiguous
```

Comment:
```suggestion
* `continuous` (`Bool`): If true, data records are contiguous. For files which are EDF-compliant but not EDF+-compliant, this field is set to `true` by default.
```

Reply:
Fixed in https://github.com/beacon-biosignals/EDF.jl/pull/18/commits/76950d031daead118bffa57c0c4ee1f9a4cb96cb.